woocommerce/woocommerce-android
# [Order Creation] UI of the search with selectable params

Closes: #9304


### Description

The PR adds the UI of the search with selectable params. It extracts and reuses compose "SearchLayout" that is also used in Product Selection Search screen

### Testing instructions

* Start order creation
* Start adding a new customer
* Notice new search for UI
* Notice that search and type selection services activity death
* Make sure that Products Selection Search is not affected by the changes

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/cae5d59d-92c2-486a-b254-00c1e4808ce6
